generators/factory/templates/factory.js

```diff
@@ -1,37 +1,27 @@
 /**
- *
+ * <%= description %>
+ * @author
  */
 
 (function () {
-
   'use strict';
 
   angular
-    .module('<%= camelCase %>Factory', [])
-    .factory('<%= camelCase %>Factory', <%= camelCase %>Factory);
+    .module('<%= module %>', [])
+    .factory('<%= factory %>', <%= factory %>);
 
   /**
-   * @param {[type]} <%= camelCase %>Factory [description]
-   * @ngInject
-   */
-  function <%= camelCase %>Factory () {
-    var data = {};
+  * Factory Definition
+  * @ngInject
+  */
+  function <%= factory %> () {
+    let data = {};
 
-    var factory = {
-      initialize: initialize,
```

Comment:
dropped the initialize function here. just thought that was too opinionated and in most cases you may not want an initialize function.